test/release/examples/benchmarks/shootout/fasta.chpl

```diff
@@ -133,7 +133,7 @@ proc randomMake(desc, a, n) {
         line_buff[i] = a[hi](nucl);
       }
     }
-    line_buff[bytes] = newline;
+    line_buff[bytes] = newline:int(8);
```

Comment:
I think all the occurrences of int(8) in fasta should become uint(8) because they all have to do with characters.  Then the casting comment in the README will also be unnecessary.